src/app/project/project-documents/project-documents.component.ts

```diff
@@ -918,6 +921,8 @@ export class ProjectDocumentsComponent implements OnInit, OnDestroy {
     window.scrollTo(0, 0);
     this.loading = true;
 
+    const cc = this.tableParams2;
```

Comment:
Seems to be a typo? Assuming this should be **tableParams**? Compile error thrown here

Reply:
oops, yup, just a debug thing, fixed...